server/models/price.py

```diff
@@ -3,29 +3,29 @@
 
 from .product import Product
 from sqlalchemy.orm import relationship
-from sqlalchemy import ForeignKey
 
 class Price(db.Model):
 
     __tablename__ = 'prices'
 
-    product = relationship('Product')
-
     primary_id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
-    product_id = db.Column(ForeignKey('product.id'), nullable=False)
-    price = db.Column(db.Integer)
+    url_id = db.Column(db.String, nullable=False)
```

Comment:
We need the database to be able to know how prices are connected to products.
If we're removing the product_id, maybe we should add the url_id to the product table and make that the foreign key.

Reply:
Ok I've connected prices to products. 

What was called url_id is just called product_id now. Its the id on Product and the foreign key on Price.

It involved changing the POST route for `/products` as you now have to provide the id when posting.

When scraping it will check if the product exists on the db, if not it will add it.

@adams-liu later today lets reconvene to make sure our changes to product aren't conflicting.